week-1/refactor/find.js

```diff
@@ -8,6 +8,7 @@ function find(list, target) {
     }
   }
   return -1;
+  
 }
 
 module.exports = find;
```

Comment:
The comment the top of the file says use for...of loop not for loop. So can you please follow the instructions?